docs/stencil-docs/page-builder/page-builder-overview.md

```diff
@@ -16,25 +16,27 @@ Page Builder is a browser-based tool that offers merchants visual editing and co
+As a theme developer, you can configure your theme's settings, which will determine how merchants can personalize your theme. For example, you can modify how merchants can customize colors, fonts, and the number of products to display per feature. For information on how to configure your theme's files, see [Theme Styles Configuration](https://developer.bigcommerce.com/stencil-docs/page-builder/theme-styles-configuration). 
 
 ## Widgets
 
 Page Builder's **Widgets** tool lets store owners create and manage page content for their stores. Merchants can use widgets in the control panel to drag and drop content such as text, images, videos, banners, carousels, and product lists. 
 
-Developers can build user interfaces for their custom [widgets](https://developer.bigcommerce.com/api-docs/storefront/widgets/widgets-overview) within the Page Builder platform using a pre-configured schema. While BigCommerce provides the setting, logic, and design, the widget author provides information about the various configurations that make up the widget. BigCommerce uses the same mechanism for all platform-provided widgets.
+Developers can build user interfaces for their custom [widgets](https://developer.bigcommerce.com/api-docs/store-management/widgets/overview#widgets) within the Page Builder platform using a pre-configured schema. While BigCommerce provides the setting, logic, and design, the widget author provides information about the various configurations that make up the widget. BigCommerce uses the same mechanism for all platform-provided widgets.
 
-New widgets are injected into the storefront using BigCommerce's [Widgets API](https://developer.bigcommerce.com/api-docs/store-management/widgets/overview). You can add custom configuration settings to your widget's Page Builder UI by including the `schema` property in the [**Create a Widget Template**](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/createwidgettemplate) request. 
+New widgets are injected into the storefront using BigCommerce's [Widgets API](https://developer.bigcommerce.com/api-docs/store-management/widgets/overview). You can add custom configuration settings to your widget's Page Builder UI by including the `schema` property in the [Create a Widget Template](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/createwidgettemplate) request. 
 
-For settings that can be passed as JSON objects into the `schema` property of the **Create a Widget Template** endpoint, see [Widgets UI Schema](https://developer.bigcommerce.com/stencil-docs/page-builder/widget-ui-schema). 
+For settings that can be passed as JSON objects into the `schema` property of the [Create a Widget Template](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/createwidgettemplate) endpoint, see [Widgets UI Schema](https://developer.bigcommerce.com/stencil-docs/page-builder/widget-ui-schema). 
```

Comment:
For settings that you can pass as JSON objects into the `schema` property of the [Create a Widget Template](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/createwidgettemplate) endpoint, see [Widgets UI Schema](https://developer.bigcommerce.com/stencil-docs/page-builder/widget-ui-schema).